src/mediaelement.js

```diff
@@ -60,6 +60,43 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
         this.setPlaybackRate(this.playbackRate);
     },
 
+    loadElt: function (elt, container, peaks) {
+        var my = this;
+
+        var media = elt;
+        media.controls = this.params.mediaControls;
+        media.autoplay = this.params.autoplay || false;
+        media.style.width = '100%';
```

Comment:
Maybe we shouldn't change the way an existing element looks? It can be done in the app's CSS.